src/main/java/seedu/address/model/person/Person.java

```diff
@@ -72,6 +74,22 @@ public BirthDate getBirthDate() {
         return Collections.unmodifiableList(contactedInfoList);
     }
 
+    public String getContactedInfoListToString() {
```

Comment:
Is it better to create a ContactedInfoList? else it might be better to write Javadocs comments since it's not as intuitive as the rest of the methods.